sos/report/plugins/containers_common.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2020 Red Hat, Inc., Pavel Moravec <pmoravec@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class ContainersCommon(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = 'Common container configs under {/etc,/usr/share}/containers'
```

Comment:
Do we care about homedirectories.
/home/*/.config/containers?

Reply:
The request to capture non-root/rootless info as a whole is covered in rhbz#1776549.  If/when that's implemented that should take care of ~/.config/containers

Reply:
Thanks.